src/core/Config.ts

```diff
@@ -2,6 +2,8 @@ import * as path from 'path';
 import * as fs from 'fs';
 import { Expression } from '@perspect3vism/ad4m';
 
+export let ad4mExecutorVersion = "0.1.32";
```

Comment:
The current package version can be get by
```
const pkg = require('./package.json');
console.log(pkg.version);
```

Reply:
awesome I was looking for something like this